### ISSUE TITLE
Add Serial Number checker built into Stacks

### ISSUE THREAD

**Issue by yabdab** [yourhead/s4#47]
An internal Serial Number checker within stacks could be HUGE. Open up demo versions of stacks.

1. Use Obj-c to verify registration based on email, serial number and product code IN Stacks.
2. Offer PHP ( or other lang )  code that builds the serial number for the dev to distribute.

**Comment by yourhead** [comment]
I agree. This will definitely happen someday. It won't make v4.0 -- but I'd really like to make it happen. not sure of the exact form it will take, but thanks for the encouragement. i need it. 😝
